src/Workleap.Extensions.MediatR.Analyzers/Internals/RuleIdentifiers.cs

```diff
@@ -14,9 +14,10 @@ internal static class RuleIdentifiers
     public const string UseNotificationHandlerOrEventHandlerSuffix = "GMDTR06";
     public const string UseGenericParameter = "GMDTR07";
     public const string ProvideCancellationToken = "GMDTR08";
-    public const string HandlersShouldNotCallHandler = "GMDTR09";
+    public const string RequestHandlersShouldNotCallHandler = "GMDTR09";
```

Comment:
COS:
Update the Roslyn analyzers to allow the use of command and queries in notification handlers.